src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -2826,6 +2826,24 @@ internal static BinaryOperationKind DeriveBinaryOperationKind(BinaryOperatorKind
         }
     }
 
+    /// <summary>
+    /// Providing implementation as OperationKind.None. This implementation is sufficient because the node doesn't survive initial binding.
```

Comment:
The class comment still says `as OperationKind.None` which is no longer the case.